src/common/SwitchRow.js

```diff
@@ -17,10 +17,11 @@ type Props = $ReadOnly<{|
 
 const componentStyles = createStyleSheet({
   container: {
-    height: 56,
+    // Minimum touch target height (and width):
+    //   https://material.io/design/usability/accessibility.html#layout-and-typography
+    minHeight: 48,
   },
   icon: {
-    marginVertical: 8,
     textAlign: 'center',
     marginLeft: 8,
     marginRight: 16,
```

Comment:
```
SwitchRow: Use container minHeight for height

We care that the touch target is at least 48px in height, and we
care that the row can expand to accommodate a larger system font
size than the standard. This seems like a transparent way to
accomplish both.
```

Hmm, I think I too lazily copy-pasted this reasoning from `NestedNavRow`. This component, `SwitchRow`, isn't a touch target across its whole surface; only the switch is a touch target.

Reply:
Yeah, I guess the touch target here is the `Switch`.

That does extend to the full height of the row after this change, so the effect is the same.

Perhaps set the `height` of the Switch to 48? And then leave the container without a height-related prop?

Reply:
> That does extend to the full height of the row after this change, so the effect is the same.

The `styles.listItem` is applying top and bottom padding of 8px, so the switch isn't extending to the full height of the row. There's still row for 8px above the top of the switch, and for 8px below the bottom.

The inspector shows that on iOS the switch has height 31 and width 51. So in the vertical direction, my device is actually giving the row a computed height of 31 + 8 + 8 = 47.

To make this row be the same height as the others (48) with common device settings, I suppose we could bump that 47 up to 48 (and handle any discrepancy on Android) by allotting more space for the switch, yeah.

Reply:
Or I could apply minHeight: 48 to the row, and replace the reasoning with something like

> To make this row be the same height as the others (48) with common device settings

rather than comments on how big touch targets should be.

Reply:
> The `styles.listItem` is applying top and bottom padding of 8px, so the switch isn't extending to the full height of the row. There's still row for 8px above the top of the switch, and for 8px below the bottom.

Hmm, I see.

Does that padding end up as part of the touch target? I think we want it to be. In that case, setting the `minHeight` of the row to 48px makes sense.

If the padding isn't part of the touch target, then since 32px isn't tall enough for the touch target we'll want to make the touch target taller.